macro/src/main/scala/SparkSchemaHelper.scala

```diff
@@ -66,16 +67,17 @@ class SparkSchemaHelperImpl(val c: Context) {
       val constructor = structType.decl(termNames.CONSTRUCTOR).asMethod
       val params = constructor.paramLists.flatten.map((param: Symbol) => {
         val name = param.name.toString
-        val isOuterOption = param.typeSignature <:< optionType
-        val isOptionNonNullableSetter = param.typeSignature <:< optionNonNullableSetter
-
-        val isOptionSetter = param.typeSignature <:< optionSetter
-        val innerType = if (isOptionSetter || isOptionNonNullableSetter) {
-          param.typeSignature.typeArgs.head.typeArgs.head
```

Comment:
Do you know what this used to be for/how it was used?

Reply:
I think it was used before and it was not removed. It's definitely not used anymore, I added a check and since it's a macro it would throw compile-time.